README.md

```diff
@@ -115,6 +115,47 @@ When you prefix a command with `appraisal`, the command is run with the
 appropriate Gemfile for that appraisal, ensuring the correct dependencies
 are used.
 
+Removing Gems using Appraisal
+-------
+
+It is common while managing multiple Gemfiles for dependencies to become deprecated and no
+longer necessary, meaning they need to be removed from the Gemfile for a specific `appraisal`.
+To do this, use the `remove_gem` declaration within the necessary `appraise` block in your
+`Appraisals` file.
+
+### Example Usage
+### Example Usage
```

Comment:
Oops, I think I did my change suggestion wrong! I don't think we wanted the double `### Example Usage`

Reply:
😆  Fixed this, I should have looked at the suggestion closer before applying it. Thank you for catching it.